app/frontend/src/views/CDOGS.vue

```diff
@@ -1,5 +1,25 @@
 <template>
   <v-container class="secure">
+    <v-alert
+      class="mt-5 mb-5 pl-7 alert"
+      type="warning"
+      icon="mdi-alert"
+      border="left"
+      color="#d9a300"
```

Comment:
I thought this color attribute was supposed to set the background color and you wouldnt need the last css rule.

Reply:
This one here is for the thicker border/accent on the left.